styleguide.config.js

```diff
@@ -25,82 +26,69 @@ module.exports = {
-    path.join(__dirname, 'styleguide/content/content.css'),
     path.join(__dirname, 'styleguide/utils/include-msw.js'),
     path.join(__dirname, 'styleguide/utils/console-error.js'),
   ],
-  sections: [
-    {
-      name: 'Overview',
-      content: path.join(__dirname, 'styleguide/content/overview/overview.md'),
-    },
-    {
-      name: 'Components',
-      components: './src/components/**/*.jsx',
-    },
-    {
-      name: 'Hooks',
-      components: './src/hooks/**/*.jsx',
-    },
-    {
-      name: 'Icon Library',
-      content: './styleguide/content/icons/index.md',
-    },
-    {
-      name: 'Guidelines',
-      sections: [
-        {
-          name: 'Accessibility',
-          content: path.join(__dirname, 'styleguide/content/guidelines/accessibility.md'),
-        },
-        {
-          name: 'Contribute',
-          content: path.join(__dirname, 'docs/contributing.md'),
-        },
-      ],
-    },
-    {
-      name: 'Brand Identity',
-      sections: [
-        {
-          name: 'Colors',
-          content: path.join(__dirname, 'styleguide/content/brand-identity/colors.md'),
-        },
-        {
-          name: 'Spacing',
-          content: path.join(__dirname, 'styleguide/content/brand-identity/spacing.md'),
-        },
-        {
-          name: 'Typography',
-          content: path.join(__dirname, 'styleguide/content/brand-identity/typography.md'),
-        },
-      ],
-    },
-    {
-      name: 'Design Patterns',
-      sections: [{
-        name: 'Basic Page Patterns',
-        content: path.join(__dirname, 'pages/design-patterns/basic-page-spacing.md'),
-      }, {
-        name: 'Page Patterns',
-        content: path.join(__dirname, 'pages/design-patterns/pages/index.md'),
-        sections: [{
-          name: 'Table Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/table-type.md'),
-        }, {
-          name: 'Form Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/form-type.md'),
-        }, {
-          name: 'Form/Table Hybrid Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/hybrid-type.md'),
-        }, {
-          name: 'Project View Page Type',
-          content: path.join(__dirname, 'pages/design-patterns/pages/project-view-type.md'),
-        }],
-      }],
-    },
-  ],
-  skipComponentsWithoutExample: true,
+  sections: [{
+    name: 'README',
+    content: path.join(__dirname, './README.md'),
+  }, {
+    name: 'Living Codebase',
+    href: 'https://github.com/mavenlink/design-system',
+  }, {
+    name: 'Living Style Guide',
+    href: 'https://www.notion.so/Mavenlink-Design-System-d3118c3e2f5647d6adfc5b1599af993e',
+  }, {
+    name: 'Form Controls',
+    components: [
+      path.join(__dirname, 'src/components/form/form.jsx'),
+      path.join(__dirname, 'src/components/form-control/form-control.jsx'), // Is this private? Should it be undocumented as implementation details?
```

Comment:
I think there are definitely use cases for composing this into bigmaven specific components. If a feature team needs a new input that conforms to some specific case, but still wants to maintain accessibility and form compatibility, I think this would be really useful.

Reply:
That's definitely true. My current headspace is: 

1. Do we want to document (with examples) for every extracted component on the MDS site? 
2. A "private component" has more utility when expressed / documented in the actual usage -- for example, `NoOptions` component has little value unless seen / interacted with in the Select component variants. 
3. I think perhaps having the source code is good enough for implementation details. If someone wanted to produce a new form control, they could look at the existing components, study their structure, and produce a new one. Whether it shares code or extracts a new shared code module is up to the developer.